channels/bo.m3u

```diff
@@ -15,7 +15,5 @@ https://stmv.panel.mivideo.pro/rtv/rtv/chunklist_w849315537.m3u8
 http://streamer1.streamhost.org:1935/salive/GMIredadvenirm/chunklist_w1507762007.m3u8
 #EXTINF:-1 tvg-id="RTP.bo" tvg-name="RTP" tvg-country="BO" tvg-language="Spanish" tvg-logo="https://i.imgur.com/eTDc0wd.jpg" group-title="",RTP (720p)
 http://136.243.3.70:1935/RtpBolivia/RtpBolivia/playlist.m3u8
-#EXTINF:-1 tvg-id="TVLATINACANAL42HD.bo" tvg-name="TV LATINA CANAL 42 HD" tvg-country="BO" tvg-language="" tvg-logo="https://ya.co.ve/3VJ" group-title="",TV LATINA CANAL 42 HD
-https://master.tucableip.com/live/tvlatinamontero/tracks-v1a1/mono.m3u8
```

Comment:
Not 24/7.

If we wanna do things well : https://master.tucableip.com/live/tvlatinamontero/playlist.m3u8